docs/site/paredit.md

```diff
@@ -128,6 +128,8 @@ You can choose to disable all default key bindings by configuring `calva.paredit
 
 You can relax how Paredit's shortcuts replace VS Code built in shortcuts a bit by setting `calva.paredit.hijackVSCodeDefaults` to `false`.
 
+You can have the kill commands copy the deleted code to the clipboard by setting `calva.paredit.killAlsoCutsToKeyboard` to `true`.
+
```

Comment:
This doesn't really belong under the sub header of **About the Keyboard Shortcuts**.

Since this PR is to some extent obsoleting the **!!! Note** on line 81. I suggest moving that note to the bottom of the **Editing** commands and reword it (well, replace the text with the text you have here, which is excellent).

Reply:
Moved it and updated the wording.